### PR TITLE
Feature: Email Service

### PR DESCRIPTION
**Pull request by nikolaichub**
Sending email support by [@nestjs-modules/mailer](https://www.npmjs.com/package/@nestjs-modules/mailer) with an email [Handlebars](https://handlebarsjs.com/) templates.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -2,12 +2,13 @@ SECRET=
 
 PORT=4455
 
-TELEGRAM_BOT_TOKEN=
```
Comment (just-hoshinon) [inline .env.example:5]: Bot token must be in the env file

File: .env.example [hunk 2]
```diff
@@ -2,12 +2,13 @@ SECRET=
...
-TELEGRAM_CHAT_ID=
-TELEGRAM_PROXY_URL=127.0.0.1:4455
```
Comment (just-hoshinon) [inline .env.example:7]: Rename this field to TELEGRAM_BOT_API_URL

File: src/v2/email/EmailModule.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+@Module({
+  imports: [
+    MailerModule.forRoot({
+      transport: {
+        host: 'smtp.gmail.com',
+        secure: false,
+        auth: {
+          user: 'fictadvisor@gmail.com',
+          pass: 'vmmennbumbzodlzt',
```
Comment (just-hoshinon) [inline src/v2/email/EmailModule.ts:17]: Use mail configuration for such purposes

File: src/v2/email/EmailModule.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        dir: join(__dirname, 'templates'),
+        adapter: new HandlebarsAdapter(),
+        options: {
+          strict: true,
+        },
+      },
+    }),
+  ],
+  providers: [EmailService],
+  exports: [EmailService], // export for DI
```
Comment (just-hoshinon) [inline src/v2/email/EmailModule.ts:33]: No need for this comment

File: src/v2/email/EmailService.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+export class EmailService {
+  constructor(private mailerService: MailerService) {}
+
+  async sendEmail({ to, subject, message, link }: EmailOptions) {
+    await this.mailerService.sendMail({
+      to,
+      subject,
+      template: './confirmation',
+      context: {
+        // filling curly brackets confirmation.hbs with content
```
Comment (just-hoshinon) [inline src/v2/email/EmailService.ts:16]: Where is the confirmation.hbs file?

File: src/v2/email/EmailService.ts [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import { MailerService } from '@nestjs-modules/mailer';
+import { Injectable } from '@nestjs/common';
+
+import { EmailOptions } from './dto/EmailDTO';
+
+@Injectable()
+export class EmailService {
+  constructor(private mailerService: MailerService) {}
+
+  async sendEmail({ to, subject, message, link }: EmailOptions) {
```
Comment (just-hoshinon) [inline src/v2/email/EmailService.ts:10]: How to send message not by confirmation template?

**Review by just-hoshinon — CHANGES_REQUESTED** [review]

File: src/v2/email/EmailService.ts (hunk 5 above)
Comment (nikolaichub) [reply src/v2/email/EmailService.ts:16]: Changed  to `template.hbs` - default template for email with message and link fields. Located in `src/v2/email/templates`

File: src/v2/email/EmailService.ts (hunk 6 above)
Comment (nikolaichub) [reply src/v2/email/EmailService.ts:10]: File changed to `template.hbs`. Provide `message` and `link` as context, so `.hbs` could paste this data into a template

**Review by just-hoshinon — APPROVED** [review]

**Pull request merged**